skycultures/Book_of_Fixed_Stars_English/constellation_names.eng.fab

```diff
@@ -0,0 +1,52 @@
+001	"Ursa Minor"	_("Ursa Minor")

```

Comment:
Please use the IAU abbreviations (UMi, UMa, ...) as short name where possible, or new 3-letter abbrevs like Arg, Hrs, FsA, FsB. There is a display option for "Short" where numbers are not good.

Reply:
done